src/prepack-standalone.js

```diff
@@ -46,6 +90,7 @@ export function prepack(code: string, options: Options = defaultOptions, errorHa
   return serialized;
 }
 
+/* deprecated: pelase use prepackString instead. */
```

Comment:
I don't think we should have public APIs that promise more than they can deliver. I have no objection to removing the deprecated comment once the function is actually implemented.